cmake/googletest/common.cmake

```diff
@@ -7,10 +7,14 @@ function(googletest_get_populate)
     GIT_TAG        release-1.10.0
     )
 
-    FetchContent_GetProperties(googletest)
-    if(NOT googletest_POPULATED)
-        FetchContent_Populate(googletest)
-        add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
+    if (TARGET gtest)
```

Comment:
I changed things around so get_populate only fetches GTest, but I had to continue using `if (TARGET gtest)` instead of `find_package(GTest)` as the second option makes the LGTM build fail.

Reply:
Fair enough for me